### PR TITLE
FunctionCommentSniff - Fatal error: str_repeat()

### PR DESCRIPTION
**Pull request by NickDickinsonWilde**
Fixes https://www.drupal.org/project/coder/issues/3212527

### CODE REVIEW

File: tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed [hunk 1]
```diff
@@ -552,3 +552,21 @@ function test38($a, $b) {
 function test39(mixed $arg) {
   return $arg;
 }
+
+/**
+ * Test multiline comments with excess spaces in the @param annotation..
```
Comment (klausi) [inline tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: double dot here?

Comment (NickDickinsonWilde) [reply tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: accidental, fixing.

File: tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed [hunk 2]
```diff
@@ -552,3 +552,21 @@ function test38($a, $b) {
 function test39(mixed $arg) {
   return $arg;
 }
+
+/**
+ * Test multiline comments with excess spaces in the @param annotation.
```
Comment (klausi) [inline tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: the @ sign should also be removed here

Comment (NickDickinsonWilde) [reply tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: Fixed. That passed on my local as it was - I presume I mis-called phpunit somehow on that - any known easy way to mess that up?

Comment (klausi) [reply tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: not sure how that could go wrong - now the coding standards are failing, I feel a bit bad for you now :-|

But after that I think it should really work.

Comment (jonathan1055) [reply tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: @NickDickinsonWilde I expect you know, but the coding standards used in the Coder module are a bit different from the rest of Drupal. What command were you using to the check the standards? If running locally then your phpunit will probably not check any standards.

Comment (NickDickinsonWilde) [reply tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed]: ah bother. I ran phpcs from a big project on that specific path. That's embarrassing. 
Thanks!

**Pull request merged**